Raspberry Pi and Orange Pi 3 support enhancement

asxtree opened the pull request:
Fixes #88

Changes:
- Added build targets for Orange Pi 3 arm64 Skybian image.
- Added build targets for Raspberry Pi arm Skybian image
- Changed Debian Stretch to Debian Buster support for Orange Pi Prime
- Refactor build file and removed duplicates

Does this change need to mentioned in CHANGELOG.md?

No

Asgaror committed:
Update README.md

Add storage paths of the images

Asgaror committed:
Update README.md

layout fix

jdknives committed:
Merge pull request #95 from skycoin/develop

Develop

jdknives committed:
Update migrate-single-binary.sh

asxtree committed:
Update

asxtree committed:
Update 2

asxtree committed:
Update build.sh

asxtree committed:
Update build.sh

asxtree committed:
Update build.sh

asxtree committed:
Update build.sh

asxtree committed:
Update build.sh

asxtree committed:
Update build.sh

asxtree committed:
Update build.sh

asxtree committed:
Update build.sh

asxtree committed:
Update build.sh

asxtree committed:
Update build.sh

asxtree committed:
Update build.conf

asxtree committed:
Update build.sh

asxtree committed:
Update skyraspbian-firstrun

asxtree committed:
Update build.sh

asxtree committed:
Update build.sh

asxtree committed:
Update build.sh

asxtree committed:
Update build.sh

asxtree committed:
Update build.sh

asxtree committed:
Update build.sh

asxtree committed:
Update build.sh

asxtree committed:
Update build.sh

asxtree committed:
Update build.sh

asxtree committed:
Update build.sh

asxtree committed:
Update build.conf

asxtree committed:
Update

asxtree committed:
Update build.sh

asxtree committed:
Update Makefile

asxtree committed:
Update build.sh

asxtree committed:
Clean up and last update to be tested before PR

asxtree committed:
Final update, ready for PR

asxtree committed:
Update CHANGELOG.md

i-hate-nicknames commented:
Looks good, great job! Just a couple of comments:
1. skybian-firstrun and skyraspbian-firstrun seem very similar, can we somehow combine them, or abstract out what is the same into a different file?
2. It looks that currently `build-skybian-img` target creates all three kinds of images? I think usually user would want one particular image, depending on the kind of board they got.

asxtree committed:
Update build

asxtree committed:
Merge branch 'dev' of https://github.com/asxtree/skybian into dev

asxtree committed:
Update build.sh

asxtree commented:
Updated skybian-firstrun and you can build the image by defining the board variable in the command like

make build-skybian-img BOARD=RPI 

For an image that will work on raspberry.
Currently I don’t know if there’s any requirement to build other images then these. For this i think we should rethink a little the build

jdknives commented:
@asxtree good job. The image works well on OPi3. I am encountering issues with RPi image but that may be my board because I encountered them before while others reported it working. @Asgaror once you approve we can merge and pre-release.

asxtree committed:
Refactor build

- Added raspberry arm64 image support
- Reconfigured build process to support other boards in the near future

asxtree committed:
Update Makefile

asxtree committed:
Update build.sh

asxtree committed:
Defined default build

asxtree committed:
Fix SkyRaspbian build

asxtree committed:
Final update

Added support for raspberry pi zero w and fixed all the issues encountered in building the images for raspberry pi

asxtree committed:
Fix LAN IP setting for Raspberry

asxtree committed:
Added time and date set
